deploy/openshift/manifests/00-contrail-08-operator.yaml

```diff
@@ -37,7 +37,7 @@ spec:
       containers:
         - name: contrail-operator
           # Replace this with the built image name
-          image: hub.juniper.net/contrail-nightly/contrail-operator:<CONTRAIL_VERSION>
+          image: <CONTRAIL_REGISTRY>/contrail-operator:<CONTRAIL_VERSION>-ubi
           imagePullPolicy: Always
```

Comment:
Can you remove changes in this file? They are not needed in the master branch, because in the master we will be using -ubi tagged images everywhere, so <CONTRAIL_VERSION> will be equal to something like `master.1430-ubi`

Reply:
sure, btw is it problematic in R2011 to <CONTRAIL_VERSION> contain the version with -ubi preffix?

Reply:
Not really, but we did not want to change the workflow for Sarath from the QA team as he already started testing.

Reply:
OK, reverted.